chain/client/src/metrics.rs

```diff
@@ -592,6 +592,18 @@ pub(crate) static ORPHAN_CHUNK_STATE_WITNESSES_TOTAL_COUNT: Lazy<IntCounterVec>
     .unwrap()
 });
 
+pub(crate) static CHUNK_STATE_WITNESS_NETWORK_ROUNDTRIP_TIME: Lazy<HistogramVec> =
+    Lazy::new(|| {
+        try_create_histogram_vec(
+            "near_chunk_state_witness_network_roundtrip_time",
+            "Time in seconds between sending state witness through the network to chunk producer \
```

Comment:
nit: do we manually break up long strings in many places by using the `\` character?  I would not bother with it and rely on `rustfmt`.

Reply:
rustfmt would most likely do the same, if this isn't already rustfmt's doing in the first place.

Reply:
ran cargo fmt and did not do it. for now I will keep it this way and check if there is anything in fmt command we can run to break long strings for future chantes.